dns/unboundbl/src/opnsense/service/conf/actions.d/actions_unboundbl.conf

```diff
@@ -1,5 +1,5 @@
 [refresh]
-command:sh /usr/local/opnsense/scripts/OPNsense/Unboundbl/UnboundBL.sh
+command:/usr/local/opnsense/scripts/OPNsense/Unboundbl/UnboundBL.sh
```

Comment:
Nothing wrong with calling the script directly from sh, it's one 3 chars longer

Reply:
Franco will set the X bits when this is in master, no worries :)

Reply:
Lol this is what I thought, but I’m trusting @mimugmail on this one!